### DISCUSSION TITLE
Guest has not initialized the display (yet) on UTM

### DISCUSSION THREAD

**Discussion by therealm1** [utmapp/UTM#4799] [UTM Help (macOS)]
i tired to startup a windows 11 vhdx file but this message pops up every time, but when i turn on UEFI Boot it starts showing startup.nsh Shell (M1 MBA)

**Comment by LareinaCornia** [comment]
me too. have you solved it yet?

**Comment by trey-a-12** [comment]
This solved it for me:

VHDX files have had more issues in my experience. I recommend getting an ARM64 file instead. Your best site for that is [https://uupdump.net/known.php](url), where you can look for a Windows 11 ARM64 ISO package and convert the download to an ISO. I found it easier to do on a secondary Windows PC, where I downloaded and converted the file to an ISO on the PC, put it in OneDrive on the PC, then downloaded it to my Mac, using that to install instead of the VHDX file.

With the new ISO, now you can make a new VM, setting it to Virtualize Windows, select the ISO, set your settings, go through the normal installer, reboot and detach the ISO, and then go through normal Windows setup. You may have to use the Command Prompt to finish setup, enbling the "I don't have internet" option, which you can look up how to do (I forgot, sorry). After finishing setup, install and run the Windows Guest Tools, and you should be good to go!

**Comment by mgh24** [comment]
Anyone else having this issue? Any solution found?

I have been using UTM and liked it a lot, but now unable to create a new VM.  Get same message posted above. 

Thanks
